vitrina/classifiers/models.py

```diff
@@ -21,7 +21,7 @@ class Category(models.Model):
     icon = models.CharField(max_length=255, blank=True, null=True)
 
     class Meta:
-        managed = False
+        managed = True
```

Comment:
All models should be `managed = False`, database should be created from a PostgreSQL SQL dump, same thing will be used in production. PostgreSQL database will be created and updated using [PGLOADER](https://pgloader.io/).

So do not create database using Django migrate, create it using PGLOADER. Once we are done with the migration, then we will switch all models to `manage = True` and switch to  Django migrations. Until then, we have to stick with PGLOADER.

Reply:
To be more specific, we need to add same MySQL version as in production to docker-compose, then load MySQL SQL schema file into MySQL instance running on Docker, and then run PGLOADER, to sync schema and data to PostgreSQL instance running on Docker.

This way we will be able to fully test migration and switch to Django migrations only when we are running PostgreSQL on production.

We can add Django migrations for specific models on top of  PGLOADER loaded schema by removeing `managed = False` line and then creating migrations:

    ./manage.py makemigrations

And then skipping initial migration:

    ./manage.py migrate --fake-initial

This way, we can stick to the original tables from PGLOADER and add new schema changes on top, using Django migrations system. But this must be done case by case and only, when we need to change existing models.

Reply:
Follow up task: https://github.com/atviriduomenys/katalogas/issues/111